plugins/home/backstage-plugin-home-markdown/src/MarkdownCard/useGithubFile.ts

```diff
@@ -29,7 +29,7 @@ export const useGithubFile = (options: MarkdownContentProps) => {
       const octokit = new Octokit({ auth: token });
 
       const response = await octokit.request(
-        `GET /repos/{owner}/{repo}/contents/${path}`,
+        `GET /repos/${owner}/${repo}/contents/${path}`,
```

Comment:
This mixes JS template string and the templating functionality of the octokit library. Octokit uses `{something}` and expects an object with `{something: 'the-val-of-something'}`, no dollar sign like JS template strings. We should use one or the other but not both

Reply:
Similarly in other spots as well.

Reply:
ahhh I was wondering how that even worked.